iothub/service/tests/ClientWebSocketTransportTests.cs

```diff
@@ -33,9 +34,9 @@ public class ClientWebSocketTransportTests
         public static void AssembyInitialize(TestContext testcontext)
```

Comment:
There is an async version of assembly init, I think.

Reply:
I'll set this aside for later investigation